src/test/rpc/Feature_test.cpp

```diff
@@ -202,16 +203,17 @@ class Feature_test : public beast::unit_test::suite
         using namespace test::jtx;
         Env env{*this};
 
-        auto jrr = env.rpc("feature", "MultiSignReserve")[jss::result];
+        std::string const name = "MultiSignReserve";
+        auto jrr = env.rpc("feature", name)[jss::result];
         BEAST_EXPECTS(jrr[jss::status] == jss::success, "status");
         jrr.removeMember(jss::status);
         BEAST_EXPECT(jrr.size() == 1);
-        BEAST_EXPECT(
-            jrr.isMember("586480873651E106F1D6339B0C4A8945BA705A777F3F4524626FF"
-                         "1FC07EFE41D"));
+        auto const expected =
+            to_string(sha512Half(Slice(name.data(), name.size())));
+        BEAST_EXPECT(jrr.isMember(expected));
```

Comment:
There may be other places the same hash check is done, but either way, one reason an explicit hash was provided in this test was to ensure that nothing breaks or changes the feature name hashing algorithm. 

Maybe add back a `BEAST_EXPECT(expected == "<hash string>");`?

Reply:
Thank, done :)